prov/efa/src/efa_base_ep.h

```diff
@@ -83,6 +85,51 @@ extern struct fi_provider efa_prov;
 /* maximum name length for shm endpoint */
 #define EFA_SHM_NAME_MAX	   (256)
 
+#define EFA_ABI_VER_MAX_LEN 8
+
+#define EFA_WCE_CNT 1024
+
+#define EFA_EP_TYPE_IS_RDM(_info) \
+	(_info && _info->ep_attr && (_info->ep_attr->type == FI_EP_RDM))
+
+#define EFA_EP_TYPE_IS_DGRAM(_info) \
+	(_info && _info->ep_attr && (_info->ep_attr->type == FI_EP_DGRAM))
+
+#define EFA_DEF_POOL_ALIGNMENT (8)
+#define EFA_MEM_ALIGNMENT (64)
+
+#define EFA_DEF_CQ_SIZE 1024
```

Comment:
should this variable be in `efa_base_ep.h`?  (it is cq related)

Reply:
I only moved things necessary for `efa_base_ep` now